pulsar-client-cpp/CMakeLists.txt

```diff
@@ -288,6 +288,10 @@ if (NOT APPLE AND NOT MSVC)
     set(CMAKE_CXX_FLAGS_PYTHON "${CMAKE_CXX_FLAGS}")
     # Hide all non-exported symbols to avoid conflicts
     set(CMAKE_CXX_FLAGS " -fvisibility=hidden -Wl,--exclude-libs,ALL ${CMAKE_CXX_FLAGS}")
+    # Turn on color erro messages and show additional help with errors:
+    set(CMAKE_CXX_FLAGS " -fdiagnostics-show-option -fdiagnostics-color ${CMAKE_CXX_FLAGS}")
```

Comment:
This might not work with older Gcc or Clang versions.

Reply:
Ah yes, good point.  I'll find the minimum versions for those and limit them.